featuremanagement/_featuremanager.py

```diff
@@ -28,17 +28,17 @@ class FeatureManager(FeatureManagerBase):
     evaluated.
     """
 
-    def __init__(self, configuration, **kwargs):
+    def __init__(self, configuration: Mapping, **kwargs: Dict[str, Any]):
         super().__init__(configuration, **kwargs)
-        filters = [TimeWindowFilter(), TargetingFilter()] + kwargs.pop(PROVIDED_FEATURE_FILTERS, [])
+        filters = [TimeWindowFilter(), TargetingFilter()] + cast(List, kwargs.pop(PROVIDED_FEATURE_FILTERS, []))
```

Comment:
Could we do Sequence instead of List since we probably don't plan to edit it?

Reply:
Just checked, mypy doesn't like `Sequence` it doesn't allow us to add the two lists together.